### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by artemkazakov947**
It was pretty easy.

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+
+    @staticmethod
+    def days_to_weeks(days):
+        if days % 7 != 0:
+            return (days // 7) + 1
+        return days / 7
+
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(
```
Comment (IvanRamyk) [inline app/main.py:16]:
```suggestion
        return cls(
```
It's better to use the `cls` parameter to access  in the classmethods

Comment (IvanRamyk) [reply app/main.py:16]: The reasons will be discussed in the following topics

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

**Review by artemkazakov947 — COMMENTED** [review]
"return OnlineCourse was replaced with return cls"

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**